Akoebbe redhat support

alangarf opened the pull request:
Merging/adjusting @akoebbe support for Redhat PR

alangarf commented:
@akoebbe hey thanks for the PR, can you please test the changes I've made to your PR as you had overridden the base :linux check and test commands. I have no way of testing the changes I've made to your PR so I've made a new one in the hope you could confirm this works just fine.

Thanks again,
-A.

akoebbe commented:
@alangarf Thanks for the changes. I don't know Ruby, so I was taking a shot in the dark with some of this. I'll test out the changes on Monday when I'm back in the office.

alangarf commented:
Ok this PR now supports CentOS 6.6 and 7 (with my brief testing). Plus I've cleaned up and simplified the plugin in general to allow the addition of new guests to be much easier to understand.
